### ISSUE TITLE
html rendering in Buckaroo

### ISSUE THREAD

**Issue by ghost** [buckaroo-data/buckaroo#190]
### Checks

- [X] I have checked that this issue has not already been reported.

- [X] I have confirmed this bug exists on the [latest version](https://github.com/paddymul/buckaroo) of Buckaroo.

### What type of jupyter notebook were you using (VSCode notebook, google colab, Jupyter Lab, Jupyter notebook).  Select multiple if you can reproduce this in multiple environments.  If other, please add to description.

Jupyter Lab

### Reproducible example

```python
from itables import init_notebook_mode
import pandas as pd
init_notebook_mode(all_interactive=True)
s = pd.DataFrame(
    [
        "<b>bold</b>",
        "<i>italic</i>",
        '<a href="https://github.com/mwouts/itables">link</a>',
    ]
)
```

### Issue description

(all_interactive=False) in Buckaroo. 
<img width="437" alt="4" src="https://github.com/paddymul/buckaroo/assets/79505893/a89c3cf7-f2dd-4c03-aeec-670558e7ebeb">

### Expected behavior

html functionality should be rendered instead of html tags.
<img width="606" alt="3" src="https://github.com/paddymul/buckaroo/assets/79505893/cc943ea0-55e8-4aa0-8811-a6dbafde7bd3">

### Installed versions

<details>
```
buckroo 0.4.8
```
</details>

**Comment by paddymul** [comment]
This is a feature request.  I should add an html renderer, and allow that to be specified on a per column basis.  Generally I don't plan to render all strings as HTML... that could be a security risk, but I do want to allow the option.

**Comment by paddymul** [comment]
This is now available as of #250 , release 0.6.5

You need to use the custom SVGDisplayer, which renders HTML.  This isn't turned on by default, but is available

```python
import pandas as pd
import buckaroo
df = pd.DataFrame({'a':
    [
        "<b>bold</b>",
        "<i>italic</i>",
        '<a href="https://github.com/mwouts/itables">link</a>',
    ]}
)
buckaroo.BuckarooWidget(df, column_config_overrides={'a': {'displayer_args': {'displayer': 'SVGDisplayer'}}})
```
<img width="996" alt="Screenshot 2024-03-08 at 1 00 38 PM" src="https://github.com/paddymul/buckaroo/assets/40453/d2780cfe-59c8-45fe-9dfc-4d63e657ae63">

**Comment by ghost** [comment]
Is the link clickable? Doesn't look like an active clickable link in the image?
